easybuild/easyconfigs/p/PuLP/PuLP-2.8.0-foss-2023b.eb

```diff
@@ -0,0 +1,36 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+# Updated: Petr Král (INUITS)
+
+easyblock = 'PythonPackage'
+
+name = 'PuLP'
+version = '2.8.0'
+
+homepage = 'https://github.com/coin-or/pulp'
+description = """
+PuLP is an LP modeler written in Python. PuLP can generate MPS or LP files and
+call GLPK, COIN-OR CLP/CBC, CPLEX, GUROBI, MOSEK, XPRESS, CHOCO, MIPCL, SCIP to
+solve linear problems.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['4903bf96110bbab8ed2c68533f90565ebb76aa367d9e4df38e51bf727927c125']
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('GLPK', '5.0'),
+    ('Cbc', '2.10.11'),
+    # Gurobi requires a seperate license
+    # ('Gurobi', '9.5.0'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+sanity_pip_check = True
```

Comment:
`download_dep_fail`, `use_pip` and `sanity_pip_check` are all `True` by default in EB 5.0.0. As such, it's cleaner to remove these from the easyconfig (this is what the tests are currently failing on)
```suggestion
```